io.catenax.shared.recycling_strategy_certificate/3.0.0/RecyclingStrategyCertificate.ttl

```diff
@@ -0,0 +1,108 @@
+#######################################################################
+# Copyright (c) 2023, 2024 SAP SE
```

Comment:
No, you don't need to add this model again in the PR. You only need to rebase your branch with main branch. If that is done then it's fine.